WireAPI/Sources/WireAPI/Models/Authorization/DomainRedirect.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+public enum DomainRedirect: String, Decodable, Sendable {
```

Comment:
I would rather have a decodable data type close to the place of decoding, so in this case a `DomainRedirectV8` enum that is private or internal, and having the public model not be decodable. 

The reason is that this model is intended for external consumption and by using it to decode network responses we give it dual responsibility. This creates a great risk that a change to the model for the purpose of external consumption breaks the decoding for the purpose of representing network models.

Reply:
Yes, I agree. I added `DomainRedirectV8` as a decodable internal data type and use it to parse the response. `DomainRedirect` is public and non-decodable